sdk/src/auth/types.ts

```diff
@@ -15,13 +15,14 @@ export interface AuthenticationStorage {
 }
 
 export interface AuthenticationConfig {
-	autoRefresh?: boolean;
-	msRefreshBeforeExpires?: number;
+	autoRefresh: boolean;
+	msRefreshBeforeExpires: number;
+	credentials: RequestCredentials;
 	storage?: AuthenticationStorage;
 }
 
 export interface AuthenticationClient<_Schema extends object> {
-	login(email: string, password: string, options: loginOptions): Promise<AuthenticationData>;
+	login(email: string, password: string, options?: loginOptions): Promise<AuthenticationData>;
```

Comment:
A bit unrelated and nitpicking, but just realized the type `loginOptions` starts with lower case letter. Is there a specific reason for this or can we change it to a capital letter to align with all other types?

Reply:
Great find